### ISSUE TITLE
Cut off canvas on small sizes

### ISSUE THREAD

**Issue by WouS95** [mkolsv/circle-progress.vue#3]
Hi, first of all i really like this package. Very easy to use and customize.

I'm using this in my list view but on small sizes the sides get cut off. Could you please help me fix this issue? 
The size i'm using is this:
```
<CircleProgressBar :value="option.takenSpots" :max="option.maxSpots" size="25px"
 strokeWidth="15" colorFilled="#ccc" colorUnfilled="#ccc" colorBack="none" />
```
See attached image for result.
![Scherm­afbeelding 2023-11-28 om 22 57 13](https://github.com/mkolsv/circle-progress.vue/assets/9462219/2c9c6fa5-4619-4c2a-9d63-731003c3af61)
-------------------------------------

### Bonus
Also, it would be really helpful if you show default values and the units that are used in the docs. This makes it easier to understand what values are expected and how it works.

For example in the table add a column:
animationDuration | String | Animation Duration | **200ms**
strokeWidth | String | Circle Stroke width | **6px**

Again, great package!

**Comment by WouS95** [comment]
The following css fixed the issue of overflowing, however it will still be technically outside of the viewBox.

```
svg.circle-progress__svg {
    overflow: visible;
}
```

**Comment by mkolsv** [comment]
Hi @WouS95 Thank you for reporting. I just released a new package version (3.2.0) where I fixed this issue and added default props values.

Hope it helps!
